src/node.rs

```diff
@@ -97,9 +97,9 @@ impl Sprawl {
         self.forest.clear();
     }
 
-    /// Remove nodes.
-    pub fn remove(&mut self, node: Node) {
-        let id = if let Ok(id) = self.find_node(node) { id } else { return };
+    /// Remove nodes. Returns the id of the node removed.
```

Comment:
Docs could be better, but that's not your fault. If #96 is merged first, we'll need to rebase this.

Reply:
Fixed as of https://github.com/DioxusLabs/sprawl/pull/106/commits/8b227be403d33ea05dfe771b143821afd9e1a98e

Reply:
Good to know. I think we can cross that bridge when we come to it.